src/test/java/org/opendevstack/provision/controller/E2EProjectAPIControllerTest.java

```diff
@@ -795,15 +779,15 @@ public void testQuickstarterProvisionOnNewOpenProjectInclDeleteSingleComponent()
             .readValue(
                 resultProjectGetResponse.getResponse().getContentAsString(), OpenProjectData.class);
 
-    assertTrue(resultProject.getQuickstarters().size() == (currentQuickstarterSize - 1));
+    assertEquals((currentQuickstarterSize - 1), resultProject.getQuickstarters().size());
 
-    // retrieve the project thru the get endpoint - to ensure we have the right data stored
+    // retrieve the project through the get endpoint - to ensure we have the right data stored
```

Comment:
comment says exactly the same as the code express, so it does not add much value, better to remove

Reply:
done